### PR TITLE
#269 increasing timeout in golangci-lint

### PR DESCRIPTION
**Pull request by sethiyash**

### CODE REVIEW

File: .github/workflows/golangci-lint.yml [hunk 1]
```diff
@@ -17,4 +17,4 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.34
-          args: -v
+          args: -v --timeout 2m
```
Comment (danielhelfand) [inline .github/workflows/golangci-lint.yml]: You may want to bump this higher. We have 5m for kapp-controller. The golangci-lint action can be flakey some times, so probably best to be conservative.

**Review by danielhelfand — COMMENTED** [review]
Also, this should probably be added to `.golangci.yml` config instead of action. Like shown here:

https://github.com/vmware-tanzu/carvel-kapp-controller/blob/adae427d34fcfd20b56721ad09e03299ab56758f/.golangci.yml#L2

File: .github/workflows/golangci-lint.yml (hunk 1 above)
Comment (sethiyash) [reply .github/workflows/golangci-lint.yml]: Sure. Thanks

**Comment by sethiyash** [comment]
> Also, this should probably be added to `.golangci.yml` config instead of action. Like shown here:
> 
> https://github.com/vmware-tanzu/carvel-kapp-controller/blob/adae427d34fcfd20b56721ad09e03299ab56758f/.golangci.yml#L2

Ah! Got it. Thanks

**Pull request merged**